sphinx_rtd_theme/layout.html

```diff
@@ -106,6 +106,11 @@
   {%- endif %}
   {%- endblock %}
   {%- block extrahead %} {% endblock %}
+
+  <link
+  rel="stylesheet"
+  href="https://cdn.jsdelivr.net/npm/docsearch.js@2.6.3/dist/cdn/docsearch.min.css"
```

Comment:
can it be not from CDN but rather from the theme itself? Or it looks like a security whole, since CDN is not maintained by algolia themselves.